Makefile

```diff
@@ -28,3 +33,28 @@ include build/makelib/image.mk
 fallthrough: submodules
 	@echo Initial setup complete. Running make again . . .
 	@make
+
+
+.PHONY: submodules fallthrough test-integration run
+
+# ====================================================================================
+# Special Targets
+
+define CROSSPLANE_MAKE_HELP
+Crossplane Targets:
+    cobertura          Generate a coverage report for cobertura applying exclusions on generated files.
+    reviewable         Ensure a PR is ready for review.
+    submodules         Update the submodules, such as the common build scripts.
+    run                Run crossplane locally, out-of-cluster. Useful for development.
```

Comment:
I believe only `submodules` will work out of this list here

Reply:
✂️ thanks!